<launch>
    <arg name="seq" default = "/00" />
    <arg name="param" default = "" /> <!-- param is used for tuning "$(arg param)" -->
    <arg name="data_path" default = "/home/ruanjy/Dataset/sequences" />
    <arg name="result_path" default = "$(find slamesh)/../../slamesh_result/kitti" />

    <rosparam file="$(find slamesh)/config/kitti_meshing.yaml" command="load" />
    <node pkg="slamesh" type="slamesh" name="slamesher" output="screen">
        <param name="seq"        value="$(arg seq)"/>
        <param name="file_loc_report"  value="$(arg result_path)"/>
        <param name="file_loc_dataset" value="$(arg data_path)"/>
    </node>
      <node name="rviz" pkg="rviz" type="rviz" args="-d $(find slamesh)/config/rviz/slamesh_mesh_tool.rviz" required="false" />
</launch>
